src/vm-vcpu/src/vm.rs

```diff
@@ -652,7 +664,6 @@ mod tests {
             ranges.push((GuestAddress(i * 100), 100))
         }
         let guest_memory = GuestMemoryMmap::from_ranges(&ranges).unwrap();
-
```

Comment:
Can you leave the whitespaces as they were? They are separating configuration and initialization and is making the code easier to read.